core/store/src/trie/update.rs

```diff
@@ -86,14 +73,28 @@ impl TrieUpdate {
                 return Ok(data.as_ref().map(TrieUpdateValuePtr::MemoryRef));
             }
         }
-
```

Comment:
tbh, I like this blank line -- cleary separates "here we fetch from 'memory cache'" from "lets go to the store"